cirq-web/circuit_example.ipynb

```diff
@@ -0,0 +1,142 @@
+    "# You may obtain a copy of the License at\n",
+    "#\n",
+    "# https://www.apache.org/licenses/LICENSE-2.0\n",
+    "#\n",
+    "# Unless required by applicable law or agreed to in writing, software\n",
+    "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "# See the License for the specific language governing permissions and\n",
+    "# limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "88f3b242",
+   "metadata": {},
+   "source": [
+    "### Cirq-web 3D circuit visualization"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9f4089af",
+   "metadata": {},
+   "source": [
+    "## need to update these links\n",
+    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://quantumai.google/cirq/circuits\"><img src=\"https://quantumai.google/site-assets/images/buttons/quantumai_logo_1x.png\" />View on QuantumAI</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/quantumlib/Cirq/blob/master/docs/circuits.ipynb\"><img src=\"https://quantumai.google/site-assets/images/buttons/colab_logo_1x.png\" />Run in Google Colab</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/quantumlib/Cirq/blob/master/docs/circuits.ipynb\"><img src=\"https://quantumai.google/site-assets/images/buttons/github_logo_1x.png\" />View source on GitHub</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a href=\"https://storage.googleapis.com/tensorflow_docs/Cirq/docs/circuits.ipynb\"><img src=\"https://quantumai.google/site-assets/images/buttons/download_icon_1x.png\" />Download notebook</a>\n",
+    "  </td>\n",
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e4bd9946",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "!pip install cirq-web"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fe0d0e69",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import cirq\n",
+    "qubits = [cirq.GridQubit(x, y) for x in range(10) for y in range(10)]\n",
+    "\n",
+    "cz = cirq.CX(qubits[0], qubits[1])\n",
+    "x = cirq.H(qubits[2])\n",
+    "test = cirq.H(qubits[0])\n",
+    "test2 = cirq.H(qubits[0])\n",
+    "moment = cirq.Moment([x, cz])\n",
+    "moment2 = cirq.Moment(test, cirq.Y(cirq.GridQubit(1, 2)));\n",
+    "moment3 = cirq.Moment(test2);\n",
+    "moment4 = cirq.Moment([cirq.H(cirq.GridQubit(1, 1)), cirq.CX(qubits[0], qubits[1])])\n",
+    "moment5 = cirq.Moment([cirq.I(cirq.GridQubit(1,2)), cirq.CCNOT(cirq.GridQubit(0, 0), cirq.GridQubit(0, 1), cirq.GridQubit(0, 2))])\n",
+    "moment6 = cirq.Moment([cirq.H(cirq.GridQubit(5, 6)), cirq.CX(qubits[7], qubits[8])])\n",
```

Comment:
I'd suggest using a random circuit here, much shorter and one can play with number of qubits as well as density and number of moments 

```
import cirq.testing

circuit = cirq.testing.random_circuit(cirq.GridQubit.square(4), n_moments=5, op_density=.7)

from cirq_web import Circuit3D
c3d = Circuit3D(circuit)
display(c3d)
```